### ISSUE TITLE
Stupid Chickens teleporting or Foxie got hungry.

### ISSUE THREAD

**Issue by Ladysarajane** [teagan75/Metamorphoses#3]
This may not be an issue....just ignorance on how the mod works...but I have lost a chicken. I have a field enclosed by a gate that I put my chicken in during sunny days and I put them in Foxie's house when it rains. I have found several times after doing my care with my chicken and they are clucking happily in the field, I leave the field, shut the gate and leave...Chicken in yard after gate closed. I go to another building and when I enter it (usually my house, but one time it was town hall) I heard a clucking and sure enough chicken on my head. Usually when I have them on my head...I see their legs..this time no.... One night after several evening chores..I entered my house. I thought chicken was in the yard, because it was going to be a "glorious sunny day" the next day and but away items and saved the game...didn't hear anything or see anything. When I started my game, my chicken was missing. I am suspecting Foxie of eating her because she was talking about how fast she can pluck feathers (I'm not kidding) and how it was so nice to get a nice sleep without the clucking. Also she wouldn't open her shop for a couple days. Not sure if it was a festival day or not. I had declined the chicken festival because my park wasn't built yet. 

I was almost ready for my chicken to level up. I finally bought a new one..but she too is displaying teleporting abilities. How can I be sure my chickens are safe? Yes I have had them one time be in Foxies house on the straw on the shelf and I left and chicken had teleported on my head again. Not sure how to stop this. I'll never level up my chickens. 

And just so you know...this pack is my new obsession. Totally loving it. I did add inventory tweaks to it (I hate messy chests) and so far no problems.

**Comment by teagan75** [comment]
wow I have never had an issue with chickens teleporting or staying on my head! I have played through the beginning chapters a lot doing my own testing and have gone through 2 yrs of HF with no issues. I don't know but I will see if someone else has had that issue.

I usually add Inventory Tweaks last because I hate the mod lol - which as a squad fan you are probably aware of :P

**Comment by Ladysarajane** [comment]
I couldn't believe they were doing it. I have to keep going back to the field to put her away. I'm not sure if it is something like accidentally calling them,,,,or what if that is even possible.  I didn't get any messages about the death of my chicken....and it disappeared from my Farming Manual. 

Are we suppose to use the Coop blueprints that  are like the Starter House?

**Comment by teagan75** [comment]
So strange. I haven’t placed the coop before so I will look into it and see if it would work for the HF chicken

**Comment by teagan75** [comment]
Quick question: did the chicken issue happen before or after you added Inventory Tweaks?

**Comment by Ladysarajane** [comment]
Before I added inventory tweaks. The problem seems to resolve itself a bit. I think Chicken was lonely. I hatched an egg and since I added that chick to the field the other Chicken seems to behave herself and take care of the chick. Waiting to see what happens when that chick grows up. After looking at the YT videos at the prefab chicken coops, the advanced one is not appropriate for HF. The smaller one may work..but it supposedly spawns with a minecraft chicken. The barn looks interesting though for multiple animals.

**Comment by teagan75** [comment]
That is still so strange. However, I have always had my chicken, cow and
sheep in the same pen. The chicken coop should not be able to spawn a
chicken since that is locked down. The barn is part of the quest line for
later.

**Comment by Ladysarajane** [comment]
Good to know that....Good thing you told me about the barn...as soon as I figured out how to get glowstone I was going to get that barn. Still trying to figure out how to get bronze for some of those magic quests...(to make extraction spell)

**Comment by teagan75** [comment]
bronze is an alloy so you will have to make the crusher first. I made each
chapter open ended but everything you need to progress is in each chapter. 

I will be pushing an update today by the way. Glad you are enjoying the pack!

**Comment by teagan75** [comment]
When you play through again, could you see what recipe Yulif gives? I want to see if it's random. Thanks

**Comment by Ladysarajane** [comment]
I will do that. My twitch wouldn't cooperating in downloading the pack...kept timing out. I finally was able to download it now and I will give it a start.

**Comment by Ladysarajane** [comment]
Okay...the first time I started a new world, I lost Yulif and couldn't get him back...he just poof! I waited three days....no respawn and then I gave the goddess the wood again and nothing happened. I deleted the instance and re downloaded and installed it...I think something was messed up...

Anyways new world...Yulifs first recipe was Pineapple juice recipe. So I think it is random. Love the Sparks hammers..makes it easier. Not sure if you want to leave in the blueprints for the Plains.... Easy way to exchange dirt for wasteland soil, but it does help provide minecraft flowers and makes it easier to green up your town.

**Comment by teagan75** [comment]
Pineapple juice is what I got as well from Yulif. Jade is toast, Tiberius
is dinner rolls and Danieru is butter. If you end up with the same then it
is set and I can make quests for them :D

Make sure you have enough RAM/memory dedicated more than 4 that Twitch
defaults to. I have noticed the more npcs I get the more I have issues. I
lost my MC builder three times and had to use the command to respawn her.

**Comment by Ladysarajane** [comment]
Yeah, Jade gave me toast and when I talked to her again a second later, she gave me french toast. I have 6gigs dedicated to Twitch and could add more, but I find more is not always better. Did your builder accidentally build themselves into somewhere? I have them suffocate or fall off things and die. I know MC for 1.10 is a little buggy. My first play through...I put cobble under my trees in the park and it was quite a distance from the lumberjack...put he still invaded the park to cut trees, then he couldn't replant them on the cobble. The cobble trick must be for newer MC. I see you have the mobs dropping chunks now.

**Comment by teagan75** [comment]
6 gigs should be plenty. No my builder never died and I could never recall
them so I just respawned them. I think I will probably turn off birch and
spruce saplings for the lumberjack till I get my tree farm built on my
youtube play through. Yeah the chunks are part of Agricultural Expansion
for resource crops later on.

**Comment by Ladysarajane** [comment]
Question, you gave us a field(scarecrow)  as a quest reward, do you want us to build an MC Farm? And how can they plant regular seeds? I thought that was disabled? I can't even plant melons in crop sticks now.

**Comment by teagan75** [comment]
No there is an option to make your own field but you need the scarecrow to
complete the build. It's an option only. the MC farms won't work with the
mod

**Comment by Ladysarajane** [comment]
Okay thanks! I was wondering that. So I was just confused with the description about the helper to help with the farm..but you were referring to the builder to help build a planting field via decorations tab.

**Comment by teagan75** [comment]
small update went out today. Just added scripts so you do not need to
restart. I have the Harvestcraft addon for Harvest Festival but realized
none of Pam's HC fish sold.

**Comment by Ladysarajane** [comment]
Okay....I think I have figured out something....if you are interacting with the chicken and it is flapping their wings, don't leave the area until the chicken is done flapping.....that is when they teleport.

**Comment by teagan75** [comment]
I am keeping this open because I believe this is a bug even in 1.12.2

**Comment by Ladysarajane** [comment]
Good to know that this hasn't been solved yet.
